payments-core/src/main/java/com/stripe/android/cards/StaticCardAccountRangeSource.kt

```diff
@@ -14,9 +14,9 @@ class StaticCardAccountRangeSource(
 ) : CardAccountRangeSource {
     override val loading: Flow<Boolean> = flowOf(false)
 
-    override suspend fun getAccountRange(
+    override suspend fun getAccountRanges(
         cardNumber: CardNumber.Unvalidated
-    ): AccountRange? {
-        return accountRanges.first(cardNumber)
+    ): Set<AccountRange> {
```

Comment:
Do we still want this to be a set? It seems like we're throwing a set in the middle without reason.